config/config.json

```diff
@@ -7,8 +7,8 @@
         "collection": "PokeData"
     },
     "shared_database": {
-        "username": "MLAB_USERNAME_HERE",
-        "password": "MLAB_PASSWORD_HERE",
+        "username": "admin",
+        "password": "admin",
```

Comment:
Also, there is no longer a config.json in this path. It has been renamed as config.js and is in the root directory.